db/schema.sql

```diff
@@ -0,0 +1,161 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.5.20
+-- Dumped by pg_dump version 12.4
```

Comment:
not blocking: you normally want to keep the pg_dump version and the server version in sync to avoid new syntax blocking an (re)import. I'd suggest using the same major.minor version, patch should be fine to bump.

Reply:
Good thing to keep in mind. In this case it's not an issue, as noted in [this article](https://pgolub.wordpress.com/2013/11/19/dump-and-restore-of-postgresql-version-compatibility-faq/).

> Q: Can I create dump of old server with new pg_dump?
> A: Yes, sure. All versions since 7.0 are supported now. But if you use custom format, be ready the same or higher version of pg_restore will be needed.

> Q: Can I use new output format of the pg_dump with old server version, e.g. directory output for 8.4 server?
> A: Yes, you can.

Reply:
I'd be careful with this and probably worth testing you can restore
> Q: Will pg_restore “understand” archives of newer versions of pg_dump?
A: No. The rule is simple: pg_restore will work with pg_dump output of the same major and minor version or lower, e.g. pg_restore v9.3.x will understand pg_dump output v9.3.x, 9.2.x, 9.1.x, 8.x etc. and will not work with output from versions like 9.4.x.

Reply:
Ah... yeah. Probably better to avoid potential version issues altogether I've regenerated all the files with pg_dump version 9.5.23.